drivers/SmartThings/zigbee-thermostat/fingerprints.yml

```diff
@@ -84,10 +84,15 @@ zigbeeManufacturer:
     manufacturer: Stelpro
     model: ST218
     deviceProfileName: thermostat-temperature-temperaturealarm
-  - id: "Stelpro/MaestroStat"
-    deviceLabel: Stelpro Thermostat
+  - id: "Maestro Thermostat"
+    deviceLabel: Maestro Thermostat
     manufacturer: Stelpro
-    model: MaestroStat
+    model: SMT402AD
+    deviceProfileName: thermostat-stelpro-profile
+  - id: "SAT402ZB Thermostat"
```

Comment:
The change here should be to add a full fingerprint, including adding a line to define the profile for the SAT402ZB device. 

The fingerprint file change for this PR should just be limited to adding a complete fingerprint such as this:

- id: "SAT402ZB Thermostat"
    deviceLabel: SAT402ZB Thermostat
    manufacturer: Stello
    model: HT402
    deviceProfileName: thermostat-stelpro-profile